packages/hash/frontend/src/blocks/page/createEditorView.ts

```diff
@@ -105,6 +196,14 @@ export const createEditorView = (
     },
   );
 
+  void manager.loadPage(state, getLastSavedValue()).then((tr) => {
+    view.updateState(state.apply(tr));
```

Comment:
Is there a reason we're doing this over using `view.dispatch`?

Reply:
No reason. Any advantages to `dispatch`? I would be interesting to know if I should be doing something else in 
https://github.com/hashintel/hash/blob/4db20e72149725297f5560645c1907d4ca862769/packages/hash/frontend/src/blocks/page/createEditorView.ts#L60
as well.

From what I understand, we wouldn't be touching anything other than the `state` prop here, so I chose to not dispatch

Reply:
I've updated this in 47dfa325f748dd0bde4e1541b6a19513e2de3b09

Reply:
Dispatch is standard.